packages/react-composites/tests/browser/common/chatTestHelpers.ts

```diff
@@ -35,6 +35,28 @@ export const chatTestSetup = async ({
   await Promise.all(pageLoadPromises);
 };
 
+export const chatTestSetupWithPerUserArgs = async ({
```

Comment:
To reduce duplicate code, can `chatTestSetup` (fn above this) call this function:
```ts
export const chatTestSetup = async (pages, users, serverUrl, qArgs) =>
  await chatTestSetupWithPerUserArgs(pages, users.map(user => ({ user, qArgs }), serverUrl);
```
(^code totally untested, just an impression of my thought)